doc_source/cluster-autoscaler.md

```diff
@@ -152,7 +152,14 @@ Complete the following steps to deploy the Cluster Autoscaler\. We recommend tha
      -n kube-system \
      -p '{"spec":{"template":{"metadata":{"annotations":{"cluster-autoscaler.kubernetes.io/safe-to-evict": "false"}}}}}'
    ```
-
+1. Patch the container memory limit to avoid `OOMKill` error in the future\.
+  
+   ```
+   kubectl patch deployment cluster-autoscaler \
+   -n kube-system \
+   -p '{"spec":{"template":{"spec":{"containers":[{"name":"cluster-autoscaler","resources":{"limits":{"memory":"1000Mi"}}}]}}}}'
```

Comment:
This should probably be left to configure for the customer, with perhaps a suggestion that makes it obvious it is not a 1-size fits all.